providers/redhat/package_feed.go

```diff
@@ -104,5 +105,8 @@ func (feed *Feed) ListFixedCVEs(d *wfn.Attributes, p *rpm.Package) ([]string, er
 		}
 	}
 
+	// Sort for deterministic output.
+	sort.Strings(cves)
```

Comment:
How big is the list? If it can be big, can we make it optional? Also, it can alway be done via external tooling...

Reply:
The list is small, it's a few CVEs per package. Not sure why we'd make it optional if we only have one user of this function atm and the option would be enabled :)

Sure, it can always be done by external tooling, but it's cumbersome: this list can end up being part of a JSON output, buried deep in an object. Doing this helped me ensuring that the output was remaining the same across a code change, that seems useful enough to have it built-in.